swagger.yaml

```diff
@@ -1314,6 +1314,141 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/usersSkills":
+    get:
+      operationId: listUsersWithSkills
+      summary: List Users with skills
+      description: Lists Users with skills
+      tags:
+        - UsersWithSkills
+      responses:
+        "200":
+          description: List of users with skills
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/UserSkills"
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+    post:
+      operationId: createUserWithSkills
+      summary: Post a new User with skills
+      description: Add new User with skills
```

Comment:
Again I'm a little confused about the working, if we are creating a UsersSkills entity, then lets just refer to it as UsersSkills rather than as a User with skills- which could lead to confusion